<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>slicePainter</class>
 <widget class="QWidget" name="slicePainter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>340</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>256</width>
    <height>340</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>256</width>
    <height>340</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>260</width>
     <height>340</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="vLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="slice">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>256</width>
        <height>256</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>256</width>
        <height>256</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="sliderLayout">
      <item>
       <widget class="QSlider" name="sliceSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdSlice">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="numDigits">
         <number>4</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="startGroupBox">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelText">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Bottom-most vertebra:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="bottomLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex">
          <number>4</number>
         </property>
         <item>
          <property name="text">
           <string>S5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>S4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>S3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>S2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>S1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>L5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>L4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>L3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>L2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>L1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>T12</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>T11</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>T10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>T9</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>T8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>T7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>T6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>T5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>T4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>T3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>T2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>T1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>C7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>C6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>C5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>C4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>C3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>C2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>C1</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
